Formula/httpd.rb

```diff
@@ -18,14 +18,188 @@ class Httpd < Formula
   depends_on "pcre"
 
   def install
-    # use Slackware-FHS layout as it's closest to what we want.
-    # these values cannot be passed directly to configure, unfortunately.
-    inreplace "config.layout" do |s|
-      s.gsub! "${datadir}/cgi-bin", "#{pkgshare}/cgi-bin"
-      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
-      s.gsub! "${datadir}/htdocs",  "#{pkgshare}/htdocs"
-      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
-    end
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    # fix default user/group when running as root
+    inreplace "docs/conf/httpd.conf.in", /(User|Group) daemon/, "\\1 _www"
```

Comment:
What about when run not as root? Note, this formula isn't designed to be run as root so I'm not sure this is needed.

Reply:
I believe httpd will ignore these configuration directives if it doesn't have the priviledges to change the owner/group of its child processes. I believe providing support for root (as is found in the macOS) version is important for those that would like to continue to use the priviledged ports 80 and 443.

Reply:
Let's pass on this for now and split root stuff into another PR so this one can get merged quicker.